src/network/utils/codec.rs

```diff
@@ -43,6 +44,19 @@ impl Decoder for NetworkMessagesCodec {
                 Ok(Some(raw_msg))
             }
             Err(encode::Error::Io(ref e)) if e.kind() == ErrorKind::UnexpectedEof => Ok(None),
+            Err(encode::Error::UnrecognizedNetworkCommand(cmd)) => {
+                warn!("Received unrecognized network command {}", cmd);
+
+                // Skip unrecognized message.
+                src.advance(4 + 12); // magic(4bytes) + command string(12bytes)
+                let payload_size = {
+                    let mut decoder = io::Cursor::new(&src);
+                    ReadBytesExt::read_u32::<LittleEndian>(&mut decoder)? as usize
+                };
+                src.advance(4 + 4 + payload_size); // length(4bytes) + checksum(4bytes) + payload
+
+                Ok(None)
```

Comment:
If this happens, the counter party may be executing a command that is not defined in the network protocol. In that case, I think that it is better not to skip, but to disconnect such a node.

Reply:
rust-tapyrus  still has unsupporting messages which defined in network protocol like `cmpctblock`, `feefilter`. So we can't decode these message. I made it just skip these messages so far.

Reply:
I see. please comment it.